.github/workflows/scorecard.yml

```diff
@@ -0,0 +1,135 @@
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Run secret-dependent integration tests
+on:
+  pull_request:
+  repository_dispatch:
+
+name: scorecard tests
+env:
+  IMAGE_NAME: scorecard
+  OUTPUT_PATH: output
+  REPO: https://github.com/prest/prest
+jobs:
+  scorecard-trusted:
+    runs-on: ubuntu-latest
+    if: (github.event_name == 'repository_dispatch') || (github.event_name == 'pull_request' && github.event.pull_request.head.repo.full_name == github.repository)
+    steps:
+      - id: debug-outputs
+        run: |
+          echo ${{ github.event_name }}
+          echo ${{ github.event.client_payload }}
+
+      - id: verify-sha-input
+        if: github.event_name == 'repository_dispatch'
+        run: |
+          echo \"${{ github.event.client_payload.pull_request.head.sha }}\"
+          echo \"${{ github.event.client_payload.slash_command.args.named.sha }}\"
+          SHAINPUT=$(echo ${{github.event.client_payload.slash_command.args.named.sha}} | cut -c1-7)
+          if [ ${#SHAINPUT} -le 6 ]; then echo "error::input sha not at least 7 characters long" ; exit 1
+          else echo "done"
+          fi
+          SHAHEAD=$(echo ${{github.event.client_payload.pull_request.head.sha}} | cut -c1-7)
+          echo ${#SHAINPUT}
+          echo ${#SHAHEAD}
+          if [ "${SHAHEAD}" != "${SHAINPUT}" ]; then echo "sha input from slash command does not equal the head sha" ; exit 1
+          else echo "shas are equal"
+          fi
+
+      - name: pull_request actions/checkout
+        if: github.event_name == 'pull_request'
+        uses: actions/checkout@5a4ac9002d0be2fb38bd78e4b4dbde5606d7042f # v2.3.4
+
+      - name: pull_request actions/checkout
+        if: github.event_name == 'repository_dispatch'
+        uses: actions/checkout@5a4ac9002d0be2fb38bd78e4b4dbde5606d7042f # v2.3.4
+        with:
+          ref: 'refs/pull/${{ github.event.client_payload.pull_request.number }}/merge'
+
+      - name: setup-go
+        uses: actions/setup-go@37335c7bb261b353407cff977110895fa0b4f7d8 # v2.1.3
+        with:
+          go-version: '1.16'
+
+      - name: Prepare test env
+        run: |
+            go mod download
+            go get github.com/ossf/scorecard
+            go install github.com/ossf/scorecard
+            mkdir $OUTPUT_PATH
+
+      - name: Test Blob cache
+        env:
+          GITHUB_AUTH_TOKEN: ${{ secrets.GH_AUTH_TOKEN }}
+          USE_BLOB_CACHE: true
+          BLOB_URL: ${{secrets.BUCKET_URL}}
+        run: |
+            scorecard --repo=$REPO --show-details --metadata=openssf  --format json > ./$OUTPUT_PATH/results.json
```

Comment:
I am one of the maintainers of scorecard. This will output the results.json and the code would have to validate the results of the json.

Reply:
Could you help us with this check so it makes sense to keep the scorecard in the pipeline (github action)